app/models/feed.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+class Feed < ActiveRecord::Base
+  has_and_belongs_to_many :pages
+
+  validates :name, presence: true, uniqueness: true
+  validates :url, presence: true, uniqueness: true
+
+  acts_as_url :name, url_attribute: :slug, only_when_blank: true, allow_duplicates: false
```

Comment:
I had written a Feeds controller to fill in for the Collections tumblr AJAX. I didn't commit it since I later removed that functionality altogether since tumblr currently is the feed with the most items(20) and that can be loaded at once. 
If we start caching longer feeds(since we allow any custom feed), re-introducing the AJAX through a feeds controller might make sense, however I didn't want to spend more time fleshing this out without knowing what the requirements/or even which "longer feed" we want to display.